### ISSUE TITLE
[feat] `use-state` should allow `return useState()`

### ISSUE THREAD

**Issue by fregante** [Rel1cx/eslint-react#1963]
The original rule has a carveout for returning `useState` directly: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/hook-use-state.md#rule-details

```tsx
import React from 'react';
export default function useColor() {
  // useState result is directly returned
  return React.useState();
}
```

I think `@eslint-react/use-state` should allow the same usage by default.
